### PR TITLE
[DDW-423] Finish ETC code cleanup

### PR DESCRIPTION
**Pull request by daniloprates**
This PR finishes the ETC code cleanup

Note: The second commit removes the `isAdaApi` and any other if the API is ADA.

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

File: source/common/environment.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ const environment = Object.assign({
   TEST: 'test',
   PRODUCTION: 'production',
   NETWORK: process.env.NETWORK || 'development',
-  API: process.env.API || 'ada',
+  API: 'ada',
```
Comment (nikolaglumac) [inline source/common/environment.js]: @daniloprates I think we can remove this line as it is no longer used in the code.

File: source/renderer/app/components/loading/Loading.js [hunk 2]
```diff
@@ -162,11 +161,11 @@ export default class Loading extends Component<Props, State> {
       !isConnected ? styles.connectingLogo : styles.syncingLogo,
     ]);
     const currencyLogoStyles = classNames([
-      styles[`${environment.API}-logo`],
+      styles['ada-logo'],
       !isConnected ? styles.connectingLogo : styles.syncingLogo,
     ]);
     const apiLogoStyles = classNames([
-      styles[`${environment.API}-apiLogo`],
+      styles['ada-logo'],
```
Comment (nikolaglumac) [inline source/renderer/app/components/loading/Loading.js]: @daniloprates here you used `ada-logo` and it should be `ada-apiLogo`...

File: source/renderer/app/components/profile/terms-of-use/TermsOfUseForm.js [hunk 3]
```diff
@@ -69,7 +68,7 @@ export default class TermsOfUseForm extends Component<Props, State> {
       isSubmitting ? styles.submitButtonSpinning : styles.submitButton,
     ]);
 
-    const checkboxLabel = environment.isEtcApi() ? 'checkboxLabelWithDisclaimer' : 'checkboxLabel';
+    const checkboxLabel = 'checkboxLabel';
```
Comment (nikolaglumac) [inline source/renderer/app/components/profile/terms-of-use/TermsOfUseForm.js]: @daniloprates we don't need this variable any longer - we can now use directly `messages. checkboxLabel`

File: source/renderer/app/components/static/About.js [hunk 4]
```diff
@@ -76,10 +66,10 @@ export default class About extends Component<Props> {
     const { onOpenExternalLink } = this.props;
     const {
       version, build, os,
-      API, API_VERSION,
+      API_VERSION,
     } = environment;
 
-    const apiName = intl.formatMessage(environmentSpecificMessages[API].apiName);
+    const apiName = intl.formatMessage(environmentSpecificMessages.ada.apiName);
```
Comment (nikolaglumac) [inline source/renderer/app/components/static/About.js]: @daniloprates we no longer need `environmentSpecificMessages` as we are using only ADA now. Please move all those messages under `globalMessages` (which means they will still be in the same file but just in the default export).

**Comment by nikolaglumac** [comment]
@daniloprates I have added the `WIP` tag - please remove it once the code is ready to be reviewed again. Thanks!

**Comment by daniloprates** [comment]
@nikolaglumac ✅

**Comment by nikolaglumac** [comment]
@daniloprates there are some flow issues - can you please fix those?

**Comment by nikolaglumac** [comment]
@daniloprates I have added some code style imporvements. The last part to be done in this PR is to merge the ADA specific stores to the generic ones (like stated in my review).

**Comment by nikolaglumac** [comment]
@daniloprates I have added the `WIP` tag now. Please remove it once ready to be merged again.

**Comment by daniloprates** [comment]
@nikolaglumac ✅

**Pull request merged**